database/seeds/UserTableSeeder.php

```diff
@@ -57,7 +57,7 @@ public function run()
 
             // Using first or create if someone is already an admin.
             $user = User::firstOrCreate([
-                 'id' => $contestant->id,
+                 'id' => $contestant->northstar_id,
```

Comment:
Should this be:

```php
'northstar_id' => $contestant->id,
```

Since $contestant is a record from Northstar and it's creating a record into Gladiator?